docs/http_api.rst

```diff
@@ -880,14 +881,20 @@ Users
     :<json str last_name: The (optional) last name of the user.
     :<json str email: The email address of the user.
     :<json str password: The password for the user.
-    :<json bool admin: True if the user is an admin user.
+    :<json bool admin:
+        (optional) True if the user is an admin user. Defaults to False.
+    :<json bool active:
+        (optional) False if the user is inactive. Inactive users cannot have
+        tokens created, and permissions are also inactive. They do not show
+        up in any users listing. Defaults to True.
```

Comment:
In Numi's API, boolean properties start with `is_` or `has_`. Would be cool if we could keep the consistency, but not very important.